providers/realtimeregister/realtimeregisterProvider.go

```diff
@@ -0,0 +1,334 @@
+package realtimeregister
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/StackExchange/dnscontrol/v4/models"
+	"github.com/StackExchange/dnscontrol/v4/pkg/diff2"
+	"github.com/StackExchange/dnscontrol/v4/providers"
+	"github.com/miekg/dns/dnsutil"
+	"golang.org/x/exp/slices"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+/*
+Realtime Register DNS provider
+
+Info required in `creds.json`:
+  - apikey
+  - premium: (0 for BASIC or 1 for PREMIUM)
+
+Additional settings available in `creds.json`:
+  - sandbox (set to 1 to use the sandbox API from realtime register)
+*/
+
+var features = providers.DocumentationNotes{
+	providers.CanAutoDNSSEC:          providers.Can(),
+	providers.CanGetZones:            providers.Can(),
+	providers.CanUseAlias:            providers.Can(),
+	providers.CanUseCAA:              providers.Can(),
```

Comment:
Question: Does realtimeregister.com support [DHCID](https://docs.dnscontrol.org/language-reference/domain-modifiers/dhcid)?

```suggestion
	providers.CanUseCAA:              providers.Can(),
	providers.CanUseDHCID:            providers.Cannot(),
```